### PR TITLE
adds the resource label events api to the gitlab client

### PR DESCRIPTION
**Pull request by whoward**
I noticed that the resource label events api was not implemented and I have to use this in a script of ours.  I have completed the full API here and added tests which seem to conform to your style (please let me know if it is missing anything).

### CODE REVIEW

File: lib/gitlab/client/resource_label_events.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    # Gets a list of all label events for a single epic.
+    #
+    # @example
+    #   Gitlab.epic_label_events(5, 42)
+    #
+    # @param  [Integer, String] project The ID or name of a project.
+    # @param  [Integer] epic_id The ID of an epic.
+    # @return [Array<Gitlab::ObjectifiedHash>]
+    def epic_label_events(project, epic_id)
+      get("/projects/#{url_encode project}/epics/#{epic_id}/resource_label_events")
```
[the remark this answers is not in the document]
Comment (whoward) [reply lib/gitlab/client/resource_label_events.rb]: thanks - i must have missed that detail (we don't have/use epics on our instance so I couldn't test).  I have fixed this.

File: lib/gitlab/client/resource_label_events.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    #
+    # @example
+    #   Gitlab.epic_label_event(5, 42, 1)
+    #
+    # @param  [Integer, String] project The ID or name of a project.
+    # @param  [Integer] epic_id The ID of an epic.
+    # @param  [Integer] id The ID of a label event.
+    # @return Gitlab::ObjectifiedHash
+    def epic_label_event(project, epic_id, id)
+      get("/projects/#{url_encode project}/epics/#{epic_id}/resource_label_events/#{id}")
```
[the remark this answers is not in the document]
Comment (whoward) [reply lib/gitlab/client/resource_label_events.rb]: fixed

**Comment by NARKOZ** [comment]
Thank you ❤️

**Comment by whoward** [comment]
you're welcome :)

**Pull request merged**